src/libraries/Common/tests/System/Collections/IList.Generic.Tests.cs

```diff
@@ -630,7 +630,7 @@ public void IList_Generic_CurrentAtEnd_AfterAdd(int count)
                     {
                         collection.Add(CreateT(seed++));
 
-                        if (Enumerator_Current_UndefinedOperation_Throws)
+                        if (count == 0 ? Enumerator_Empty_Current_UndefinedOperation_Throws : Enumerator_Current_UndefinedOperation_Throws)
```

Comment:
Do we need tests validating that empty collections return the same instance on `GetEnumerator()` calls?

Reply:
Added